internal/pkg/vm/internal/vmcontext/validation_test.go

```diff
@@ -0,0 +1,24 @@
+package vmcontext
+
+import (
+	"testing"
+
+	vsuites "github.com/filecoin-project/chain-validation/suites"
+)
+
+func TestChainValidationSuite(t *testing.T) {
+	t.SkipNow()
```

Comment:
This is where the chain-validation tests get wired up. None of them pass since they`applyMessages` doesn't implement gas tracking, skipping them for now.